Subject: Rename binary from 'calls' to 'gnome-calls'
From: Evangelos Ribeiro Tzaras <devrtz@fortysixandtwo.eu>

* src/meson.build: Renamed executable
* data/sm.puri.Calls.desktop: Changed Exec and TryExec to reflect the new name
--- a/src/meson.build
+++ b/src/meson.build
@@ -118,7 +118,7 @@
 )
 
 executable (
-  'calls',
+  'gnome-calls',
   calls_sources, calls_enum_sources, calls_resources,
   wl_proto_sources, wayland_sources, 'main.c',
   dependencies : calls_deps,
--- a/data/sm.puri.Calls.desktop
+++ b/data/sm.puri.Calls.desktop
@@ -6,8 +6,8 @@
 Keywords=Telephone;Call;Phone;Dial;Dialer;PTSN;
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
 Icon=sm.puri.Calls
-TryExec=calls
-Exec=calls %u
+TryExec=gnome-calls
+Exec=gnome-calls %u
 Type=Application
 StartupNotify=true
 Terminal=false
--- a/data/sm.puri.Calls-daemon.desktop
+++ b/data/sm.puri.Calls-daemon.desktop
@@ -6,8 +6,8 @@
 Keywords=Telephone;Call;Phone;Dial;Dialer;PTSN;
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
 Icon=sm.puri.Calls
-TryExec=calls
-Exec=calls --daemon
+TryExec=gnome-calls
+Exec=gnome-calls --daemon
 Type=Application
 StartupNotify=true
 NoDisplay=true
